src/components/About/Overview/index.js

```diff
@@ -41,6 +41,10 @@ const Button = styled(_Button)`
   margin: 4px;
 `;
 
+const Link = styled(_Link)`
```

Comment:
Put styled-component which is used multiple times into a single file.

Reply:
@akshatnitd changes have been done